docs/concepts/_category_.json

```diff
@@ -0,0 +1,9 @@
+{
+    "label": "Concepts",
```

Comment:
not a strong opinion, wonder if `specification` or `architecture` make more sense.

Reply:
good point Binbin.  I want to follow a existing pattern used by other open source docs. E.g. k8s have a concepts (then there is architecture inside the concepts folder). GK docs have both concept and architecture. For Ratify, which content falls under Concept? and which content falls under architecture?

Reply:
https://kubernetes.io/docs/concepts/

Reply:
thanks for the reference, I'd like to follow the K8s as well. I feel definition and specification are under concept, while the design and implementation details could fall under architecture. In terms of this PR, I think it's good to keep them under concept.

Reply:
thanks Binbin. They can be moved around as we add more content